src/main/java/seedu/duke/manager/OrderManager.java

```diff
@@ -5,42 +5,66 @@
 
 import java.util.ArrayList;
 
-
+/**
+ * OrderManager.
+ */
 public class OrderManager {
-    private final ArrayList<Order> orders;
-    private final DishManager dishManager = new DishManager();
-    private final ArrayList<Dish> dishes;
+    private ArrayList<Order> orders;
+    private DishManager dishManager = new DishManager();
+    private ArrayList<Dish> dishes;
 
     public OrderManager() {
         this.orders = new ArrayList<Order>();
         this.dishes = dishManager.getDishes();
     }
 
+    /**
+     * Add the created order to the order list.
+     *
+     * @param order the created order.
+     */
     public void addOrder(Order order) {
         this.orders.add(order);
     }
 
-    public int addDishToOrder(Object inputObj) {
+    /**
+     * Add dish by index to the order.
+     *
+     * @param dishIdx the index of the dish.
+     * @return size of the order list.
+     */
+    public int addDishToOrder(int dishIdx) {
```

Comment:
Same here, if you are using dish index, need to geet the same `DishManager`(and its `dishes`) .